prometheus_client/gc_collector.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python
+
+from __future__ import unicode_literals
+
+import gc as gc_module
+import os
+import time
+
+from . import core
+
+class GCCollector(object):
+    """Collector for Garbage collection statistics."""
+    def __init__(self, registry=core.REGISTRY, gc=gc_module):
+        if not hasattr(gc, 'callbacks'):
+            return
+
+        _prefix = 'python_gc_'
```

Comment:
I prefer to have all the name together, makes it easier to grep

Reply:
do you mean that I should drop the prefix and just have the complete names in the registration?

Reply:
Yes, have complete names rather than building them up.